mne/viz/tests/test_epochs.py

```diff
@@ -104,9 +104,10 @@ def test_plot_epochs():
     """Test epoch plotting"""
     import matplotlib.pyplot as plt
     epochs = _get_epochs()
-    epochs.plot(scalings=None, fig_title='Epochs', trellis=False)
+    epochs.plot(scalings=None, title='Epochs', trellis=False)
     plt.close('all')
-    epochs[0].plot(picks=[0, 2, 3], scalings=None, trellis=False)
+    fig = epochs[0].plot(picks=[0, 2, 3], scalings=None, trellis=False)
+    fig.canvas.key_press_event('escape')
```

Comment:
Yes there is a test. On line 142 the number of epochs should be one less after the click on axis.